amora/models.py

```diff
@@ -204,6 +206,13 @@ def model_file_path(cls) -> Path:
     def unique_name(cls) -> str:
         return str(cls.__table__)
 
+    @classmethod
+    def owner(cls) -> str:
+        if model_owner := cls.__model_config__.owner:
```

Comment:
```suggestion
        if model_owner := cls.__model_config__.owner:
```
Primeira vez que vejo `:=` num PR